app/Console/Commands/CrawlData.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Exceptions\InvalidRaceEmailException;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+use Services\RaceMonitor\RaceMonitor\RaceMonitor;
+
+class CrawlData extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'email:send';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Email the top 10 table Race and ATR Index data, email:send {email?}';
```

Comment:
Putting the signature in the description is a nice touch, but isn't required. If you run `php artisan email:send --help` the output will display the parameters etc

Reply:
Understood! Do you want me to remove it?